ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -64,8 +93,53 @@ function WorkloadCvesOverviewPage() {
         }
     );
 
+    const defaultStorage: VulnMgmtLocalStorage = {
+        preferences: {
+            defaultFilters: {
+                Severity: isFixabilityFiltersEnabled ? ['Critical', 'Important'] : [],
+                Fixable: isFixabilityFiltersEnabled ? ['Fixable'] : [],
+            },
+        },
+    } as const;
+
+    const [storedValue, setStoredValue] = useLocalStorage(
+        'vulnerabilityManagement',
+        defaultStorage,
+        isVulnMgmtLocalStorage
+    );
+    // Until the ROX_VULN_MGMT_FIXABILITY_FILTERS feature flag is removed, we need to used empty default filters
+    // as a fallback
+    const localStorageValue = isFixabilityFiltersEnabled ? storedValue : defaultStorage;
+
     const pagination = useURLPagination(20);
 
+    function updateDefaultFilters(values: DefaultFilters) {
+        pagination.setPage(1);
+        setStoredValue({ preferences: { defaultFilters: values } });
+        setSearchFilter(
+            mergeDefaultAndLocalFilters(
+                localStorageValue.preferences.defaultFilters,
+                values,
+                searchFilter
+            )
+        );
+    }
+
+    // When the page is initially visited and no local filters are applied, apply the default filters.
+    //
+    // Note that this _does not_ take into account a direct navigation via the left navigation when the user
+    // is already on the page. This is because we do not distinguish between navigation via the
+    // sidebar and e.g. clearing the page filters.
+    useEffect(() => {
+        if (
+            isFixabilityFiltersEnabled &&
+            isEmpty(querySearchFilter) &&
+            localStorageValue.preferences.defaultFilters
+        ) {
+            setSearchFilter(localStorageValue.preferences.defaultFilters, 'replace');
+        }
+    }, []);
```

Comment:
I'm not aware of a workaround for this other than attaching an `onClick` listener to the left navigation item that resets the default filters.

If we think it is important enough I can add this as well.